uint/benches/bigint.rs

```diff
@@ -88,6 +88,42 @@ fn u512_mul_small(b: &mut Bencher) {
     });
 }
 
+#[bench]
+fn u512_rem(b: &mut Bencher) {
+    b.iter(|| {
+        let one = U512([8326634216714383706, 15837136097609390493, 13004317189126203332, 7031796866963419685, 12767554894655550452, 16333049135534778834, 140317443000293558, 598963]);
+        let two = U512([11707750893627518758, 17679501210898117940, 2472932874039724966, 11177683849610900539, 2096410819092764509, 8483673822214032535, 36306297304129857, 3453]);
+        let n = black_box(one % two);
```

Comment:
Here the compiler says that `n` is unused which is correct. You can just simplify it to `black_box(one % two);`. The same counts for the other new benchmark functions.